common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON

```diff
@@ -0,0 +1,1256 @@
+					"request": {
+						"method": "POST",
+						"header": [
+							{
+								"key": "Content-Type",
+								"value": "application/json"
+							},
+							{
+								"key": "X-chkp-sid",
+								"value": "{{session}}"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\"name\": \"<account_name>\", \"regions\": [\"eu-west-1\", \"us-east-2\"], \"credentials_file\": \"AWS_credentials_file_name\", \"access_key\": \"<access_key>\", \"secret_key\": \"<secret_key>\", \"sts_role\": \"<sts_role>\", \"sta_external_id\": \"<sts_external_id>\", \"scan_gateways\": \"<scan_gateways>\", \"scan_vpn\": \"<scan_vpn>\", \"scan_load_balancers\": \"<scan_load_balancers>\", \"access_subnets\": \"<access_sunbets>\", \"deletion_tolerance\": \"<deletion_tolerance>\"}",
+							"options": {
+								"raw": {
+									"language": "json"
+								}
+							}
+						},
+						"url": {
+							"raw": "{{server}}/v1.8/cme-api/v1.2/accounts/aws",
+							"host": [
+								"{{server}}"
+							],
+							"path": [
+								"v1.8",
+								"cme-api",
+								"v1.2",
+								"accounts",
+								"aws"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "PUT set AWS account",
+					"request": {
+						"method": "PUT",
+						"header": [
+							{
+								"key": "Content-Type",
+								"value": "application/json"
+							},
+							{
+								"key": "X-chkp-sid",
+								"value": "{{session}}"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\"regions\": [\"eu-west-1\", \"us-east-2\"], \"credentials_file\": \"AWS_credentials_file_name\"}",
+							"options": {
+								"raw": {
+									"language": "json"
+								}
+							}
+						},
+						"url": {
+							"raw": "{{server}}/v1.8/cme-api/v1.2/accounts/aws/<account_name>",
+							"host": [
+								"{{server}}"
+							],
+							"path": [
+								"v1.8",
+								"cme-api",
+								"v1.2",
+								"accounts",
+								"aws",
+								"<account_name>"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "POST add AWS gw configuration",
+					"request": {
+						"method": "POST",
+						"header": [
+							{
+								"key": "X-chkp-sid",
+								"value": "{{session}}",
+								"type": "text"
+							},
+							{
+								"key": "Content-Type",
+								"value": "application/json",
+								"type": "text"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\"name\": \"<gwConfigurations_name>\", \"base64_sic_key\": \"<encoded_base64_sic_key>\", \"version\": \"<gw_version>\", \"policy\": \"<policy_name>\", \"blades\": {\"ips\":true, \"anti-bot\":true, \"url-filtering\": true, \"https-inspection\": true}, \"related_account\": \"<associated_account>\", \"vpn_domain\": \"<vpn_domain>\", \"vpn_community\": \"<vpn_community>\", \"deployment_type\": \"<TGW>\", \"tgw_static_routes\": \"<10.0.0.0/16,10.100.0.0/16>\", \"tgw_spoke_routes\": \"<192.168.100.0/24,192.168.200.0/24>\"}"

```

Comment:
Please change to "raw": "{\"name\": \"<**gwConfiguration_name**>\", \"base64_sic_key\": \"<encoded_base64_sic_key>\", \"version\": \"<gw_version>\", \"policy\": \"<policy_name>\", \"blades\": {\"ips\":true, \"anti-bot\":true, \"url-filtering\": true, \"https-inspection\": true}, \"related_account\": \"<associated_account>\", \"vpn_domain\": \"<vpn_domain>\", \"vpn_community\": \"<vpn_community>\", \"deployment_type\": \"TGW\", \"tgw_static_routes\": **[\"10.0.0.0/16\", \"10.100.0.0/16\"**], \"tgw_spoke_routes\": **[\"192.168.100.0/24\", \"192.168.200.0/24\"]**}"